pyproject.toml

```diff
@@ -23,7 +23,11 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
 requires-python = ">=3.8"
-dependencies = []
+dependencies = [
+    "packaging>=14.1",
+    "pydantic>=1",
+    "scrapy>=2",
```

Comment:
I wonder if it'd be an issue if we want to install this library to https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy